fennel/internal_lib/schema/schema.py

```diff
@@ -224,6 +226,45 @@ def get_python_type_from_pd(type):
     return type
 
 
+def convert_dtype_to_arrow_type(dtype: schema_proto.DataType) -> pa.DataType:
```

Comment:
are there tests for all the conversion functions including this?

Reply:
added tests